### DISCUSSION TITLE
YouChat 开发请求

### DISCUSSION THREAD

**Discussion by youtehub** [viopsa233/AI-testers-Forum#15] [Q&A]
[YouChat](https://you.com/search?q=who+are+you&tbm=youchat&cfr=chat),另外一个搜索引擎，有老哥区看看能否训化呢，用了一下感觉还不错，似乎与chatgpt不同源，使用体验也还可以

**Comment by viopsa233** [comment]
我这里体验挺垃圾的🤔

**Comment by lbls888** [comment]
太烂了，不太建议使用，我直接都扔一边了，还扬言可以免费使用他们的API，这么烂怎么用啊
